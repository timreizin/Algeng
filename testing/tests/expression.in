5+4*7+2
(12+1*3-2)*2
-4+2-3-4*1
-(33*2*2)
(4+2)+38*3*(2+4+(47+23*(3+2)+1))
