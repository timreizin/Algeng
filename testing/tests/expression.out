65
74
-9
-132
47124
